openwisp_notifications/tests/test_notifications.py

```diff
@@ -102,15 +102,49 @@ def test_email_not_present(self):
         self.assertEqual(len(mail.outbox), 0)
 
     def test_group_recipient(self):
-        operator = super()._get_operator()
+        operator = self._get_operator()
+        user = self._create_user(
+            username='user', email='user@user.com', first_name='User', last_name='user'
+        )
         op_group = Group.objects.get(name='Operator')
         op_group.user_set.add(operator)
+        op_group.user_set.add(user)
         op_group.save()
         self.notification_options.update({'recipient': op_group})
+        recipients = (operator, user)
+
+        # Test for group with no target object
+        n = self._create_notification().pop()
+        if n[0] is notify_handler:
```

Comment:
please why are you checking if n[0] is `notify_handler`? i thought `self._create_notification` calls notify_handler which returns a list of `notifications created. Please help me get on track

Reply:
Send returns the following:

> Return a list of tuple pairs [(receiver, response), ... ].

[From Django Docs](https://docs.djangoproject.com/en/3.0/_modules/django/dispatch/dispatcher/#Signal.send)

Reply:
I see. thanks